packages/suite/src/reducers/firmware/firmwareReducer.ts

```diff
@@ -52,10 +51,15 @@ const firmwareUpdate = (state: FirmwareUpdateState = initialState, action: Actio
         switch (action.type) {
             case FIRMWARE.SET_UPDATE_STATUS:
                 draft.status = action.payload;
+                if (action.payload === 'started') {
+                    draft.error = undefined;
```

Comment:
delete draft.error

Reply:
Just checked that it really works 
![image](https://user-images.githubusercontent.com/30367552/96465623-745ad100-1229-11eb-80ce-8c3d44897b8d.png)